src/ifElse.test.js

```diff
@@ -2,10 +1,57 @@
+/* eslint quotes: ["off"] */
+"use strict";
 
-describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+describe("ifElse", () => {
+  const { ifElse } = require("./ifElse");
 
-  it('should ', () => {
+  it("should be declared", () => {
+    expect(ifElse).toBeInstanceOf(Function);
+  });
+
+  it("should call condition cb", () => {
+    const cond = jest.fn(() => {});
+    const cb1 = jest.fn(() => {});
+    const cb2 = jest.fn(() => {});
+
+    ifElse(cond, cb1, cb2);
+    expect(cond).toHaveBeenCalled();
+  });
+
+  it("should run first cb if condition return true", () => {
+    const cond = jest.fn(() => true);
+    const cb1 = jest.fn(() => {});
+    const cb2 = jest.fn(() => {});
+
+    ifElse(cond, cb1, cb2);
+    expect(cb1).toHaveBeenCalled();
+  });
+
+  it("should run second cb if condition return false", () => {
+    const cond = jest.fn(() => false);
+    const cb1 = jest.fn(() => {});
+    const cb2 = jest.fn(() => {});
 
+    ifElse(cond, cb1, cb2);
+    expect(cb2).toHaveBeenCalled();
   });
 
-  // write tests here
+  it("should return undefined if cond return false", () => {
+    const cond = jest.fn(() => false);
+    const cb1 = jest.fn(() => {});
+    const cb2 = jest.fn(() => {});
+
+    const result = ifElse(cond, cb1, cb2);
+
+    expect(result).toBe(undefined);
+  });
+
+  it("should return undefined if cond return true", () => {
+    const cond = jest.fn(() => true);
+    const cb1 = jest.fn(() => {});
+    const cb2 = jest.fn(() => {});
+
+    const result = ifElse(cond, cb1, cb2);
+
+    expect(result).toBe(undefined);
```

Comment:
Similar to the previous test, this one expects `ifElse` to return `undefined` when the condition returns true. Verify if this matches the expected behavior of the `ifElse` function. If the function should return the result of the executed callback, consider revising this test.